### PR TITLE
Copy .htaccess file when initilizing multisite on test container.

### PR DESCRIPTION
**Pull request by peterwilsoncc**
### Description of the Change

Ensures the htaccess file is copied when the test container's Docker environment is converted to multisite.

Closes #

### How to test the Change

1. Start docker
2. `npm run env:destroy` to reset the environment
3. `npm run env:start`
4. Visit http://localhost/second/ to ensure you can access the second site on the test environment.

### Changelog Entry

> Fixed - Copy the htaccess when initilizing the E2E test environment.

### Credits

Props @peterwilsoncc, @theskinnyghost, @dkotter.

### Checklist:

- [x] I agree to follow this project's [**Code of Conduct**](https://github.com/10up/.github/blob/trunk/CODE_OF_CONDUCT.md).
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [ ] All new and existing tests pass.

### CODE REVIEW

**Review by theskinnyghost — DISMISSED** [review]

**Review by dkotter — COMMENTED** [review]
Curious on why this is needed? I know we already have an npm script that copies the htaccess file before tests are run (see https://github.com/10up/distributor/blob/develop/.github/workflows/cypress.yml#L100 and https://github.com/10up/distributor/blob/develop/package.json#L63) so wondering if that's not working?

**Comment by peterwilsoncc** [comment]
@dkotter It's for running the tests locally rather than in CI. It allows devs to run `npm run env:start; npm run cypress:run` locally.

At the moment `env:start` does the conversion to multisite but misses the step for copying the htaccess file.

**Comment by dkotter** [comment]
> @dkotter It's for running the tests locally rather than in CI. It allows devs to run `npm run env:start; npm run cypress:run` locally.
> 
> At the moment `env:start` does the conversion to multisite but misses the step for copying the htaccess file.

Okay, I guess the question I have then is should we remove the copying of the htaccess file from the `cypress.yml` file then? I believe at the moment that is happening twice, once as part of the `postenv:start` command and then again in the `cypress.yml` file when it runs `npm run copy-htaccess`. I haven't tested this locally but seems like we can get rid of that second one now.

File: .github/workflows/cypress.yml [hunk 1]
```diff
@@ -99,9 +99,6 @@ jobs:
     - name: Copy .htaccess
       run: npm run copy-htaccess
 
-    - name: Test
```
Comment (dkotter) [inline .github/workflows/cypress.yml:102]: Looks like the wrong step was removed. This test step should stay and I think the step above should be removed

**Review by dkotter — CHANGES_REQUESTED** [review]

File: .github/workflows/cypress.yml (hunk 1 above)
Comment (peterwilsoncc) [reply .github/workflows/cypress.yml:102]: Fixed in 9b7c613c9e14fea43efad76196010d5e7c4054d5, I force pushed the incorrect commit to oblivion to keep the commit history clean.

**Review by dkotter — APPROVED** [review]

**Pull request merged**